newsletter/backend/.dockerignore

```diff
@@ -0,0 +1,5 @@
+.idea
+.vscode
+*/**.pyc
```

Comment:
unify this with .gitignore **/*.pyc

Reply:
Done. I  made new commit with suggested changes.